keys.py

```diff
@@ -0,0 +1,10 @@
+class keys:
```

Comment:
class name is confusing. 
fllow the convension as well 

- constants in the class should be capital, e.g SCHEMA, TABLES, etc 

- Also search how to make sure in python they are constant

Reply:
ok did it